server/routes/kanban.js

```diff
@@ -1,5 +1,72 @@
 const express = require('express');
 const router = express.Router();
-const knex = require('../database/knex');
+const Card = require('../database/models/Card')
+
+function isAuthenticated(req, res, next) {
+  if (req.isAuthenticated()) { next(); }
+  else {
+    res.json({ success: false, error: 'not authenticated' });
+  }
+}
+
+router.route('/') //make card
+  .post(isAuthenticated, function (req, res) {
+    Card.forge({
+      title: req.body.title,
+      body: req.body.body,
+      priority_id: req.body.priority_id,
+      status_id: req.body.status_id,
+      created_by: req.user.id,
+      assigned_to: req.body.assigned_to
+    }).save()
+      .then(function (err) {
+        res.json({ success: true });
+      })
+      .catch(function (err) {
+        res.json({ success: false, error: err });
+      });
+  })
+
+router.route('/:id')
+  .get(function (req, res) { //select card
+    Card.where('id', req.params.id).fetch()
+      .then(function (card) {
+        res.json(card.attributes);
+      })
+      .catch(function (err) {
+        res.json({ success: false, error: 'card does not exist' });
+      });
+  })
+  .put(isAuthenticated, function (req, res) { //update card
+    Card.where('id', req.params.id).fetch()
```

Comment:
I believe when you set the `id` on a new `Card` object bookshelf changes the `save` method to an `UPDATE` rather than an `INSERT` automagically so this pre-fetch lookup is not necessary.